<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWindows">
   <widget class="QGroupBox" name="groupBoxProperty">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1051</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 85, 127);</string>
    </property>
    <property name="title">
     <string>资产清单</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1031</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGraph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>1051</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 170, 0);
color: rgb(85, 170, 255);</string>
    </property>
    <property name="title">
     <string>资产报表</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="File">
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExport"/>
    <addaction name="actionCancel"/>
   </widget>
   <widget class="QMenu" name="Tool">
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionMD5"/>
    <addaction name="actionPortCheck"/>
    <addaction name="actionPing"/>
    <addaction name="actionDNS"/>
    <addaction name="actionPublicIP"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="Check">
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>巡检</string>
    </property>
    <addaction name="actionSystemCheck"/>
    <addaction name="actionDatabaseCheck"/>
    <addaction name="actionMonitorReport"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Check"/>
   <addaction name="Tool"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProperty"/>
   <addaction name="actionAddItem"/>
   <addaction name="actionChangeItem"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>报表导出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMD5">
   <property name="text">
    <string>MD5加密</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPortCheck">
   <property name="text">
    <string>端口检测</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPing">
   <property name="text">
    <string>Ping连通性检测</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDNS">
   <property name="text">
    <string>域名解析</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPublicIP">
   <property name="text">
    <string>公网IP地址解析</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionSystemCheck">
   <property name="text">
    <string>系统巡检</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDatabaseCheck">
   <property name="text">
    <string>数据库巡检</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMonitorReport">
   <property name="text">
    <string>监控报表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="text">
    <string>资产清单</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这可以查看当前中心资产清单！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionAddItem">
   <property name="text">
    <string>信息录入</string>
   </property>
   <property name="toolTip">
    <string>信息录入</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionChangeItem">
   <property name="text">
    <string>信息变更</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
